packages/react-components/react-provider/src/components/FluentProvider/useFluentProviderThemeStyleTag.ts

```diff
@@ -1,63 +1,67 @@
-import { useId, usePrevious } from '@fluentui/react-utilities';
+import { useId, useIsomorphicLayoutEffect } from '@fluentui/react-utilities';
 import * as React from 'react';
 import type { FluentProviderState } from './FluentProvider.types';
 import { fluentProviderClassNames } from './useFluentProviderStyles';
 
+const useInsertionEffect = (React as never)['useInsertion' + 'Effect']
```

Comment:
Doesn't this work by just doing

```suggestion
const useInsertionEffect = (React as never)['useInsertionEffect']
```

What's the need for the `+` sign?

Reply:
Nope, https://github.com/microsoft/fluentui/issues/23625#issuecomment-1193376468 (see P.S. part)